### DISCUSSION TITLE
HDF5 support not implemented?

### DISCUSSION THREAD

**Discussion by bilderbuchi** [OpenModelica/OpenModelica#10583] [Q&A] [answered]
I just tried to load a v7.3 `.mat` file I created into a `CombiTable1Ds`, and I'm getting
> No HDF5 support which is required to read the v7.3 MAT file "xyz.mat"

I noticed that there's a `libhdf5-0.dll` in OM's `bin` directory, so the relevant library seems to be present. Do I need to add some option or flag or somesuch to activate support? 
I went through the options, and searched in the user guide for "hdf5", but no dice. Am I missing something?

I'm asking, because I created that file from a vanilla HDF5 file using some Python libraries (no Matlab here), so I can't rule out the file being weird in some way.

I'd be fine with using ExternData instead of the MSL built-in, but sadly this is also broken on Windows: #4356.

**Comment by casella** [comment]
No idea. @mahge, @adrpo, @AnHeuermann any idea what is libhdf5 for?

**Reply by AnHeuermann** [reply]
Judging from the source code documentation of  `Mat_Open` (which prints this warning) we should support MAT files version 5 and 7.3.
For the 7.3 version it needs to be compiled with `HAVE_HDF5`, but I think this is currently disabled?
@mahge added a CMakeLists.txt in there, so probably only needs to define `HAVE_HDF5`.

**Reply by AnHeuermann** [reply]
There is also a configure in ModelicaExternalC, but it only sets `HDF5`. Probably a quick fix if the lib is already installed.

**Reply by mahge** [reply]
Sorry. Somehow I missed this. I will check and get back to you.

**Reply by bilderbuchi** [reply, marked as answer]
Tracking this in #10880
